pom.xml

```diff
@@ -72,9 +72,9 @@
         <maven-jar-plugin.version>3.1.0</maven-jar-plugin.version>
         <sundrio.version>0.19.0</sundrio.version>
 
-        <fabric8.kubernetes-client.version>4.6.4</fabric8.kubernetes-client.version>
-        <fabric8.openshift-client.version>4.6.4</fabric8.openshift-client.version>
-        <fabric8.kubernetes-model.version>4.6.4</fabric8.kubernetes-model.version>
+        <fabric8.kubernetes-client.version>4.10.0</fabric8.kubernetes-client.version>
```

Comment:
worth getting 4.10.1 ?

looks like there was a regression in the 4.10.0 release https://github.com/fabric8io/kubernetes-client/issues/2189 that was addressed in a new release today https://github.com/fabric8io/kubernetes-client/releases/tag/v4.10.1

Reply:
Yeah, in my local branch I have bumped to 4.10.1 but there is some build issue (you can take a look at travis build logs).